lib/Doctrine/MongoDB/Aggregation/Builder.php

```diff
@@ -42,7 +42,7 @@ class Builder
     /**
      * @var Stage[]
      */
-    private $stages;
+    private $stages = [];
```

Comment:
Depending on how long this PR will stay open, could we add this change separately in the next release? Calling `getPipeline` on a new Aggregation Builder will break because of the non-array `$stages` in the `array_map` call.